extensions/replication/ReplicationConfigValidator.js

```diff
@@ -36,22 +51,22 @@ const joiSchema = joi.object({
             }).when('type', { is: 'role', then: joi.required() }),
         }).required(),
     },
-    destination: {
+    destination: joi.object({
         transport: transportJoi,
-        auth: joi.object({
-            type: joi.alternatives().try('account', 'role', 'service')
-                .required(),
-            account: joi.string()
-                .when('type', { is: 'account', then: joi.required() }),
-            vault: joi.object({
-                host: joi.string().optional(),
-                port: joi.number().greater(0).optional(),
-                adminPort: joi.number().greater(0).optional(),
-                adminCredentialsFile: joi.string().optional(),
-            }),
-        }).required(),
+        auth: destinationAuthJoi.optional(),
+        sites: joi.object().pattern(
+            joi.string(), // site name
+            joi.object({
+                transport: transportJoi,
+                auth: destinationAuthJoi.required(),
```

Comment:
- since it is required, `destination.auth` will never be used as default
- does it make sense to specify a destination with the same transport but a dedicated auth?

Reply:
I made it optional, that way we could specify a different transport if needed. The validation for when the global auth is not specified will be done in `_validatePerSiteDestinationConfig`